integration/raft/cft_test.go

```diff
@@ -531,6 +532,11 @@ var _ = Describe("EndToEnd Crash Fault Tolerance", func() {
 				Expect(err).NotTo(HaveOccurred())
 
 				By("updating the config for " + o.Name)
+				if i == leaderIndex {
```

Comment:
Actually, I think an even simpler (yet still correct) solution is to continue polling the orderer for the config block until it receives it successfully instead of exiting early when we see SERVICE_UNAVAILABLE.

Reply:
Works for me.